ConfigSpace/forbidden.py

```diff
@@ -210,8 +214,7 @@ def is_forbidden(self, instantiated_hyperparameters, strict=True):
         # outcomes
         evaluations = []
         for component in self.components:
-            e = component.is_forbidden(instantiated_hyperparameters,
-                                       strict=strict)
+            e = component.is_forbidden(instantiated_hyperparameters)
```

Comment:
type of strict is bool so its value should be 'true' or 'false' and not 'strict'